overlord/assertstate/assertmgr.go

```diff
@@ -101,12 +102,96 @@ func DB(s *state.State) asserts.RODatabase {
 	return cachedDB(s)
 }
 
-// Add the given assertion to the system assertiond database. Readding the current revision is a no-op.
+// Add adds the given assertion to the system assertion database.
```

Comment:
Wait, `Add` adds? are you sure?

snark aside: i think you can (and should) drop the “adds”.